mattermost-plugin/webapp/src/index.tsx

```diff
@@ -172,7 +172,14 @@ export default class Plugin {
             }
 
             if (rudderKey !== '') {
-                rudderAnalytics.load(rudderKey, rudderUrl)
+                const rudderCfg = {} as {setCookieDomain: string}
+                if (siteURL && siteURL !== '') {
```

Comment:
maybe worth adding a simple url validation check here? WDYT?
```suggestion
                if (siteURL && siteURL !== '' && /^(http|https):\/\/[^ "]+$/.test(siteURL)) {
```

Reply:
SiteURL is being received from MM config. I don't think it makes much sense to add a second validation here.

Reply:
ok, thanks for the clarification!